scheds/rust/scx_layered/src/main.rs

```diff
@@ -1752,8 +1752,8 @@ impl<'a> Scheduler<'a> {
                     }
 
                     let util = if util < 0.01 { 0.0 } else { util };
-                    let low = (util / util_range.1).ceil() as usize;
-                    let high = ((util / util_range.0).floor() as usize).max(low);
+                    let low = (util / util_range.0).ceil() as usize;
+                    let high = ((util / util_range.1).floor() as usize).max(low);
```

Comment:
Let me do some more digging then, maybe the bug is elsewhere.

Reply:
I don't super follow this but looking at the surrounding code, it's all rust, I'd say pull it out to a standalone function w/ a bunch of args, call that from within the impl function (so it's easy to test), and write a test for the pulled out function w/ a handful of cases so it's clear it's doing what it should (because this matters a lot from the htop screenshots).